p/themes/BlueLagoon/BlueLagoon.css

```diff
@@ -564,6 +565,25 @@ a.btn {
 	color: #0090FF
 }
 
+/*=== Scrollbar */
+
+#sidebar {
+	scrollbar-color: rgba(0, 0, 0, 0.0) rgba(0, 0, 0, 0.0);
+}
+
+#sidebar:hover {
+	scrollbar-color: rgba(255, 255, 255, 0.3) rgba(0, 0, 0, 0.0);
+}
+
+/*Note: These have to be split, because Gecko ignores the line when combined, as of 4 December 2018*/
```

Comment:
"These"? You mean Blink would do something with `rgba(255, , 0.3)` or something?

Reply:
There is another selector `#sidebar:hover::-webkit-scrollbar-thumb {}` with the same rule `#sidebar:hover::-webkit-scrollbar-thumb` just below.
It would be natural but does not work. The formulation of the comment could maybe be improved. Feel free to edit :-)